tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "baseUrl": "src",
     "outDir": "build/dist",
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
IIRC, this is something that should be set to `commonjs` but VSCode will update it to `esnext` for some reason. Does that sound familiar?

Reply:
It does. I'll revert it. Thanks.

Reply:
Fixed. Thanks.